how-it-works/5_governance/00_tokenomics.subpage.md

```diff
@@ -0,0 +1,55 @@
+
+### Governance and voting rewards
+
+Anyone can participate in the governance of the IC by staking ICP tokens in so-called neurons. Neuron holders can vote on proposals, which are suggestions on how the IC should be changed. The neurons’ voting power for decision making is proportional to the number of ICP staked inside. However, the voting power also depends on some other neuron characteristics, in particular, for how long tokens are staked. For example, a neuron can boost its voting power by 100% by setting the time staked to the maximum of 8 years. The increased voting power for neurons with longer staking time creates an incentive to vote on proposals with the aim of driving decisions that maximize the value of their staked ICP over the long term.
+
+For participation in governance, neurons receive voting rewards which can be converted into newly minted ICP. Every day the IC calculates a voting reward pot according to a schedule, which it then divides among all eligible neurons according to their relative voting power. The schedule for voting rewards is designed to incentivize early adoption: Initially at genesis, 10% of the total supply of ICP is distributed in voting rewards on an annualized basis. Over the course of eight years, this number falls to 5%.
+
+Since, the daily reward amount is independent of the overall amount of staked ICP in the system, participants receive larger rewards if overall staking and participation in governance decreases. This mechanism creates a natural incentive to stake ICP and participate in governance. As of November 2022, 266M ICP is staked, corresponding to 54% of the total supply. A significant part of staked ICP, namely 123M ICP (i.e., 46%), is staked for the maximum time of 8 years expressing the long-term commitment of these stakers.
+
+Neurons can be configured to vote automatically by following the votes of other neurons, in an advanced form of “liquid democracy.” Neurons that vote automatically still receive their full share of the voting reward, as they enable the IC community to reach decisions securely and quickly.
+
+The following graph depicts annualized voting rewards as a function of the staking time as of November 1, 2022. For current estimates of annualized voting rewards, refer to the IC Dashboard’s [Governance page](https://dashboard.internetcomputer.org/governance).
+
+![](/img/how-it-works/voting_rewards.png)
```

Comment:
image does not exist

Reply:
@sesi200 : I have uploaded it before via a separate PR, see here https://github.com/dfinity/portal/pull/694/files